test/support/utils.ex

```diff
@@ -114,6 +114,27 @@ defmodule NextLS.Support.Utils do
     end
   end
 
+  defmacro assert_opened(
```

Comment:
Not the best name, but I need to notify the server so that it loads the text for a certain URI into the documents assigns. Other things that I could do is to load the files preemptively with the initialization step. Another way would be to just expect the client to send the text as well? I think this is the cleanest though, just a little overhead in the tests.

Reply:
seems fine, can you rename to `did_open` ?

and can you actually submit this as a separate PR, and add it to all the tests? then i can merge that and i won't have to approve your CI test runs every time you push